cards/faq-accordion/component.js

```diff
@@ -21,7 +21,7 @@ class faq_accordionCardComponent extends BaseCard['faq-accordion'] {
       // If the card's details are longer than a certain character count, you can truncate the
       // text. A toggle will be supplied that can show or hide the truncated text.
       // showMoreDetails: {
-      //   showMoreLimit: null, // Character count limit
+      //   truncatedDetails: profile.answer ? ANSWERS.formatRichText(profile.answer, "answer", linkTarget, 500) : null, // The truncated rich text
       //   showMoreText: '', // Label when toggle will show truncated text
```

Comment:
If showMoreLimit is still an option, maybe we should still include the commented out showMoreLimit option in the cards? Or is this what product wanted

Reply:
Rose said to get rid of it

Reply:
I also think that it's confusing having both